lib/racecar/cli.rb

```diff
@@ -58,6 +59,11 @@ def run
         $stderr.puts "=> Ctrl-C to shutdown consumer"
       end
 
+      if config.liveness_probe_file_path
+        $stderr.puts "=> Liveness probe enabled"
+        config.install_liveness_probe
+      end
```

Comment:
I know this review has dragged out a bit, but there's one last change I think we should make 😄 

I think it would make sense to have a separate `liveness_probe_enabled` boolean config flag, and to auto-generate the file path if not explicitly set, using [Tempfile](https://ruby-doc.org/stdlib-2.5.3/libdoc/tempfile/rdoc/Tempfile.html). Since we don't need to synchronize the path between the Kubernetes probe config and Racecar now that the latter has an abstraction in place, it's probably better to just auto-generate tempfiles for the liveness data, and not need users to worry about it.

Reply:
Looks good! And yes, you're right – I forgot we had multiple processes in the mix here 🤦

Reply:
The default path should probably be `"#{Dir.tmpdir}/racecar-liveness` since we want to use the proper system TMPDIR. Otherwise we might get into trouble on e.g. kpods that use read-only file systems for most paths.

Reply:
Yes! I was looking through the docs for `Dir.tmpdir`! Couldn't remember how to get that.

Adding now.

Reply:
You have to `require "tmpdir"` to get the method. Very well hidden!

Reply:
Sneaky!